### DISCUSSION TITLE
End of Life

### DISCUSSION THREAD

**Discussion by lookme2** [inventree/InvenTree#5215] [Q&A] [answered]
How do you guys handle EOL for a part?  I've been adding a Parameter for it but there is really no way to "Mark" the part as such.

**Comment by SchrodingersGat** [comment, marked as answer]
@lookme2 what about the "active" flag? You could mark a part as inactive to signify EOL

**Reply by lookme2** [reply]
That's what I've been doing.  Just wanted to make sure I was not missing something else.  Thanks!
